README.md

```diff
@@ -87,19 +82,12 @@ $ cd $GOPATH/src/github.com/ipfs/go-ipfs
 $ make install
 ```
 
-Tip:
-
-If the `make install` hangs while fetching dependencies and if you are
-running a daemon locally, ensure that it is on at least version 0.4.0-dev. If
-you don't have a build that is new enough, run the `make install` with your
-daemon turned off and the deps will be fetched through the ipfs.io gateways.
```

Comment:
Is this tip no longer needed? it's not findable, the install process should warn about it or something.